app/uk/gov/hmrc/customsservicestatusfrontend/models/ServiceStatus.scala

```diff
@@ -47,7 +47,6 @@ object State {
 }
 
 case class CustomsServiceStatus(
-  id:             String,
```

Comment:
I think we should put the `id` in the model here, as the backend has it and will be sending it over. Ultimately, it will be used.